tests/test_document.py

```diff
@@ -276,5 +276,14 @@ def test_from_identifier(self):
         assert mit.url == 'http://spdx.org/licenses/MIT'
 
 
+class TestException(TestCase):
+
+    def test_exception_list(self):
+        assert EXCEPTION_MAP["Linux Syscall Note"] == "Linux-syscall-note"
```

Comment:
nit pikcing... do you mind using single quotes for strings to stay consistent with the rest of the code?

Reply:
Pushed 0125f07